src/OnlineSales/Program.cs

```diff
@@ -131,6 +134,58 @@ public static void Main(string[] args)
         app.Run();
     }
 
+    private static void ConfigureImportSizeLimit(WebApplicationBuilder builder)
+    {
+        string fileSize;
+        string measurement;
+        int size;
+        // With default size
+        long sizeInByte = 30 * 1024 * 1024;
```

Comment:
Why do we need that default value if we re-calculate it anyway? There is no way how it could stay with default value because on line 163 you throw an exception if the configuration property is not defined. Plus, the .NET already has it is own default value and what I would do - I would check if ApiSettings:MaxImportSize exists and will apply it if it was set. If not - we do not need to do anything which mens the default .NET limits will be applied.